lib/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,35 @@
+export interface ToastProps {
```

Comment:
Would it make sense in Toast that we lock down the color and icon combinations? 
Success = check_circle, etc?

Reply:
Would probably make sense. Need more information from the designer on that.